src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const TOTAL__PRICE = days * 40;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return TOTAL__PRICE - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    return TOTAL__PRICE - LONG_TERM_DISCOUNT;
+  }
```

Comment:
you would have less code if evaluated the longTerm first
```suggestion
  if (days >= LONG_TERM) {
    return TOTAL__PRICE - LONG_TERM_DISCOUNT;
  }

  if (days >= SHORT_TERM) {
    return TOTAL__PRICE - SHORT_TERM_DISCOUNT;
  }

```